editor/modes/visual-editor/block.js

```diff
@@ -200,18 +207,27 @@ class VisualEditorBlock extends wp.element.Component {
 			>
 				{ ( showUI || isHovered ) && <BlockMover uid={ block.uid } /> }
 				{ showUI &&
-					<div className="editor-visual-editor__block-controls">
-						<BlockSwitcher uid={ block.uid } />
-						{ !! settings.controls && (
-							<Toolbar
-								controls={ settings.controls.map( ( control ) => ( {
-									...control,
-									onClick: () => control.onClick( block.attributes, this.setAttributes ),
-									isActive: control.isActive ? control.isActive( block.attributes ) : false,
-								} ) ) } />
-						) }
-						<Slot name="Formatting.Toolbar" />
-					</div>
+					<CSSTransitionGroup
+						transitionName={ { appear: 'appear-animation', appearActive: 'is-appearing' } }
```

Comment:
> Maybe transitionName="is"

I don't like this option because it's misleading too, it's kind of a hack

Using an empty string is not possible (still a JS error).

What about `{ appear: 'is-appearing', appearActive: 'is-appearing-active' }` and style `is-appearing-active`?